skymp5-functions-lib/src/logic/SweetPieGameModeListener.ts

```diff
@@ -183,6 +185,17 @@ export class SweetPieGameModeListener implements GameModeListener {
     }
   }
 
+  onPlayerLeave(actorId: number) {
+    const round = getPlayerCurrentRound(this.rounds, actorId);
+    if (round && round.players?.size === 0) {
```

Comment:
Condition is always false. If you are in the round there is at least 1 player in the round. it's you. right?